data/islamabad-infobox.txt

```diff
@@ -0,0 +1,83 @@
+{{Infobox settlement
```

Comment:
The one used is the full one. If I remember well, the full one was the conflictive one. And since wikijs is passing the whole thing to the parser I used the full one for the test as well.

I have removed the unused file.